schema@v1.1.1/schema.json

```
@@ -52,7 +52,7 @@
     "idString": {
       "type": "string",
       "minLength": 1,
-      "pattern": "^[a-z][ A-Za-z0-9]*$"
+      "pattern": "^[a-z][ A-Za-z]*[_0-9]*$"
```

```suggestion
      "pattern": "^[a-z][A-Za-z]*[_0-9]*$"
```

We haven't supported spaces in identifiers for quite a while.